manifests/redis.pp

```diff
@@ -0,0 +1,13 @@
+# deploys redis server on 127.0.0.1
+# should use https://github.com/voxpupuli/puppet-redis for now
+# @api private
+class pulpcore::redis {
+
+  include ::redis
+
+  class { '::redis':
+    bind => '127.0.0.1',
+    manage_repo => true,
```

Comment:
In general we don't want to manage repos or at least make it a parameter so we can disable it for users who manage their own repos, for example via Pulp but also downstream where we can't enable EPEL.